src/main/java/uk/gov/hmcts/reform/cpo/exception/ApiException.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.cpo.exception;
+
+@SuppressWarnings({"PMD.MissingSerialVersionUID"})
```

Comment:
Minor: Can remove suppression as pmd has been removed (won't bother polluting the rest of the PR with the same comment, but same for elsewhere too!) 
```suggestion
```

Reply:
Resolved in 7546f595ec0be3b8185b0d483b205b48f1596a01